### PR TITLE
Geomap: adding option to merge legend label and symbol

### PR DESCRIPTION
**Pull request by Garneauma**
Geomap: Adding the option to hide the symbol's description when there is only one item in the legend layer as the description is most likely the same as the label.

Changes related to WET-450

### CODE REVIEW

**Comment by Garneauma** [comment]
Pre-approved upon successful review.

File: src/plugins/deps/geomap-lib.js [hunk 1]
```diff
@@ -3546,7 +3546,12 @@ MapLegend.prototype.symbolize = function( mapLayer ) {
 	}
 
 	// append the list to the legend
-	$( "#sb_" + layerName ).html( "<ul class='list-unstyled'>" + symbolList + "</ul>" );
+	if ( $( "#sb_" + layerName ).closest( ".wb-geomap" ).hasClass( "legend-label-only" ) && symbolList.match( /<li>/g ) && symbolList.match( /<li>/g ).length === 1 ) {
+		$( "#sb_" + layerName ).html( "<ul class='list-unstyled' role='presentation'>" + symbolList + "</ul>" );
```
Comment (duboisp) [inline src/plugins/deps/geomap-lib.js:3550]: I suggest to add a comment explaining why we are setting the role presentation on the `<ul>` and `<li>` element

**Review by duboisp — CHANGES_REQUESTED** [review]
Just a little change to add a comment for some clarity in the code.

I reviewed all the code/example and tested locally, everything do work as expected and nothing to report.

Once the little comment is added, it will be good to merge.

**Review by duboisp — APPROVED** [review]
Reviewed and all good.

This is a minor change.

**Pull request merged**